packages/vsce/package.json

```diff
@@ -334,6 +334,11 @@
         "title": "Show Attributes",
         "category": "IBM CICS for Zowe Explorer"
       },
+      {
+        "command": "cics-extension-for-zowe.showRegionLogs",
+        "title": "Show Logs",
```

Comment:
```suggestion
        "title": "Show region's logs",
```

Reply:
I think I could go to "Show Region Logs" but everything's capitalised in those context menus and I'm not keen on the apostrophe (although with no good reason!)

Reply:
I've updated to `Show Region Logs`